README.md

```diff
@@ -323,6 +324,20 @@ Creates an array of unique values that are included in all given arrays.
 
 **[⬆️ back to top](#available-helpers)**
 
+#### `invoke`
```

Comment:
It totally is, but I found nothing wrong on this file, must be a bug on github’s highlighting.

-- 
Fabio Mazarotto

On 22 de março de 2016 at 00:04:33, Lauren Tan (notifications@github.com) wrote:

In README.md:

> @@ -323,6 +324,20 @@ Creates an array of unique values that are included in all given arrays.
>   
>  **[⬆️ back to top](#available-helpers)**
>   
> +#### `invoke`
Is it just me or are these additions strangely highlighted?



—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub